src/listFiles.ts

```diff
@@ -1,11 +1,11 @@
 import * as glob from "glob";
 
-export function listFiles(cwd: string, ignore: string[]) {
+export function listFiles(cwd: string, ignore: string[] = []) {
```

Comment:
Add return type `: string[]`

Reply:
so, for what it's worth, I tried to find a tslint rule that would enforce that, and I can't. However, it does implicitly know it's `string[]`, which is good. I'll keep looking for the rule config to enforce it.